docs/content/getting-started.mdx

```diff
@@ -28,8 +28,8 @@ def get_name():
 
 
 @op
-def hello(context, name: str):
-    context.log.info(f"Hello, {name}!")
+def hello(name: str):
+    get_dagster_logger().info(f"Hello, {name}!")
```

Comment:
not a fan here - this is a relatively heavy piece to add to this "lets run some code to see if you've installed dagster correctly". maybe we can just print instead of using logging at all.

Reply:
👍 - will change to print